cvat-ui/src/components/annotation-page/canvas/canvas-wrapper3D.tsx

```diff
@@ -153,34 +258,59 @@ const CanvasWrapperComponent = (props: Props): ReactElement => {
     );
 
     return (
-        <Layout.Content className='cvat-canvas3d-fullsize'>
+        <Layout.Content className='cvat-canvas3d-fullsize' id='canvas3d-container'>
             <ContextImage
                 frame={frameData}
                 contextImageHide={contextImageHide}
                 getContextImage={getContextImage}
                 loaded={loaded}
                 data={data}
             />
-
-            <div className='cvat-canvas3d-perspective'>
-                <div className='cvat-canvas-container cvat-canvas-container-overflow' ref={perspectiveView} />
-                {renderArrowGroup()}
-                {renderControlGroup()}
-            </div>
-            <div className='cvat-canvas3d-orthographic-views'>
-                <div className='cvat-canvas3d-orthographic-view cvat-canvas3d-topview'>
-                    <div className='cvat-canvas3d-header'>TOP VIEW</div>
-                    <div className='cvat-canvas3d-fullsize' ref={topView} />
-                </div>
-                <div className='cvat-canvas3d-orthographic-view cvat-canvas3d-sideview'>
-                    <div className='cvat-canvas3d-header'>SIDE VIEW</div>
-                    <div className='cvat-canvas3d-fullsize' ref={sideView} />
+            <ResizableBox
```

Comment:
Please, resolve typescript issues here and in this file

![Screenshot from 2021-03-17 11-41-07](https://user-images.githubusercontent.com/40690378/111438766-b1ae8680-8715-11eb-84da-5883fe311064.png)

Reply:
Resolved

Reply:
@manasars 

Not exactly, I see some as before (with refs)